config/routes.rb

```diff
@@ -6,4 +6,7 @@
   resources :mentees
   resources :mentorships
 
+  post 'login', to: 'login#authenticate'
+  post 'register', to: 'users#create'
+
```

Comment:
@rebeccasedgwick @leekarho we don't think we need `post 'register'` - we already have a route to the `users#create` action